scripts/prebuild.js

```diff
@@ -0,0 +1,98 @@
+const spawn = require('child_process').spawn;
+const globby = require('globby');
+const chalk = require('chalk');
+
+/** Execute the `cmd` in a new process. */
+const exec = (cmd) => {
+    return new Promise((resolve, reject) => {
+        console.log(chalk.green(`  ${cmd}`));
+        const command = spawn(cmd, [], {
+            shell: true,
+            stdio: 'inherit'
+        });
+
+        command.on('error', (err) => {
+            return reject(err);
+        });
+
+        command.on('exit', (code) => {
+            if (code !== 0) {
+                return reject(new Error('NoExitCodeZero'));
+            }
+
+            return resolve(true);
+        });
+
+    });
+};
+
+/**
+ * Searches all the `package.json` under `packages` and returns
+ * an array with the path and the content of each one.
+ * @returns { {packagePath: string, content: string}[] } The packages information
+ */
+const getPackages = async () => {
+    const packagesPaths = await globby([
+        `packages/*/package.json`,
+        `!packages/**/dist/**/*`,
+        `!packages/**/node_modules/**/*` // needed when we are inside a package like extension-vscode
```

Comment:
Now that we use `webpack` with `extension-vscode` we can probably remove `nohoist` from our root `package.json` and avoid this particular exception.

Reply:
Because of a change in another PR where we're no longer bundling the VS Code extension with webpack during test runs we'll actually still need this.